thamos/config.py

```diff
@@ -41,6 +42,77 @@
 _LOGGER = logging.getLogger(__name__)
 _THAMOS_DISABLE_TLS_WARNING = bool(int(os.getenv("THAMOS_DISABLE_TLS_WARNING", 0)))
 
+# The schema is enforcing all the options. This will make sure the right version of Thamos is
+# installed and no configuration options are silently ignored.
+_CONFIG_SCHEMA = {
```

Comment:
is the config schema something we should have as a separate file?

Reply:
Is there any functionality that would require it?

Reply:
I didn't want to expose this outside of the module. Users can use library functions for validating the schema. If you wish to have it separated, let me know. I would like to include this functionality in the next s2i release so we guarantee users use correct configuration options during the s2i build process.

Reply:
It could come in handy, but is note required..